intrinsics_tester/JIT.h

```diff
@@ -0,0 +1,162 @@
+#pragma once
+
+// Copyright (c) 2021-present The Alive2 Authors.
+// Distributed under the MIT license that can be found in the LICENSE file.
+// Version: April 7, 2022
+
+#include "llvm/ADT/StringRef.h"
+#include "llvm/ExecutionEngine/JITSymbol.h"
+#include "llvm/ExecutionEngine/Orc/CompileUtils.h"
+#include "llvm/ExecutionEngine/Orc/Core.h"
+#include "llvm/ExecutionEngine/Orc/ExecutionUtils.h"
+#include "llvm/ExecutionEngine/Orc/ExecutorProcessControl.h"
+#include "llvm/ExecutionEngine/Orc/IRCompileLayer.h"
+#include "llvm/ExecutionEngine/Orc/JITTargetMachineBuilder.h"
+#include "llvm/ExecutionEngine/Orc/RTDyldObjectLinkingLayer.h"
+#include "llvm/ExecutionEngine/SectionMemoryManager.h"
+#include "llvm/IR/DataLayout.h"
+#include "llvm/IR/LLVMContext.h"
+#include <memory>
+
+// For Tester Namespace
+#include "commandLineUtil.h"
+#include "irGenerator.h"
+#include "irWrapper.h"
+#include "x86Intrin.h"
+#include "llvm/MC/TargetRegistry.h"
+#include "llvm/Support/TargetSelect.h"
+#include "llvm/Target/TargetOptions.h"
+
+namespace llvm {
+namespace orc {
+
+class JIT {
+private:
+  std::unique_ptr<ExecutionSession> ES;
+
+  DataLayout DL;
```

Comment:
In Alive2's convention, for header files, we uses full type names with namespaces, for example please use llvm::DataLayout rather than bare DataLayout here.